doc/source/Installation.rst

```diff
@@ -32,292 +32,46 @@ also be installed:
-
-For example, to change the optimization to high, type:
-
-.. highlight:: none
-
-::
-
-    ~/grackle/src/clib $ make opt-high
-
-.. warning::
-   Compiling Grackle in single precision (with ``make precision-32``) is **not**
-   recommended. Because of the high dynamic range involved in calculating many
-   chemistry and cooling rates, running Grackle in single precision can produce
-   unreliable results. This is especially true when running with
-   :c:data:`primordial_chemistry` >= 1.
-
-Custom settings can be saved for later use by typing:
-
-.. highlight:: none
-
-::
-
-    ~/grackle/src/clib $ make save-config-<keyword>
-
-They will be saved in the **.grackle** directory in your home directory.  To 
-reload them, type:
-
-.. highlight:: none
-
-::
-
-    ~/grackle/src/clib $ make load-config-<keyword>
-
-For a list of all available make commands, type:
-
-.. highlight:: none
-
-::
-
-    ~/grackle/src/clib $ make help
-
-    ========================================================================
-       Grackle Makefile Help
-    ========================================================================
-    
-       make                Compile and generate librackle
-       make install        Copy the library somewhere
-       make help           Display this help information
-       make clean          Remove object files, executable, etc.
-       make dep            Create make dependencies in DEPEND file
-    
-       make show-version   Display revision control system branch and revision
-       make show-diff      Display local file modifications
-    
-       make help-config    Display detailed help on configuration make targets
-       make show-config    Display the configuration settings
-       make show-flags     Display specific compilation flags
-       make default        Reset the configuration to the default values
-
-4. Compile and Install
-
-To build the code, type:
-
-::
-
-    ~/grackle/src/clib $ make 
-    Updating DEPEND
-    Compiling calc_rates.F
-    Compiling cool1d_multi.F
-    ....
-    
-    Linking
-    Success!
-
-Then, to install:
-
-::
-
-    ~/grackle/src/clib $ make install
-
-5. Test your Installation
-
-Once installed, you can test your installation with the provided example to
-assure it is functioning correctly.  If something goes wrong in this process,
-check the ``out.compile`` file to see what went wrong during compilation,
-or use ``ldd`` (``otool -L`` on Mac) on your executable to determine what went 
-wrong during linking.
+.. code-block:: shell-session
 
-::
+    ~ $ git clone --recursive https://github.com/grackle-project/grackle
 
-    ~/grackle/src/clib $ cd ../example
-    ~/grackle/src/example $ make clean 
-    ~/grackle/src/example $ make 
+The presences of the ``--recursive`` flag in the above snippet instructs git to fetch additional files, containing cooling tables, from a submodule linked to the Grackle repository.
```

Comment:
I still think it's worth mentioning how to get the submodules manually. I imagine many people will forget the `--recursive` flag because they are already familiar with cloning and then not know what to do. I know for a fact people routinely search this documentation for the `git submodule update --init` line because they never remember it. Maybe just below this say that if you've already cloned without the recursive flag that you can do `git submodule update --init`?

Reply:
Can do!